XamlControlsGallery/ControlPages/SliderPage.xaml

```diff
@@ -27,21 +28,65 @@
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+        
         <local:ControlExample x:Name="Example2" HeaderText="A Slider with range and steps specified.">
-            <StackPanel x:Name="Control2" Orientation="Horizontal">
-                <Slider x:Name="Slider2" Header="Control header" Width="200" Minimum="500" Maximum="1000"
-                        LargeChange="100" StepFrequency="10" SmallChange="10" Value="800"/>
-            </StackPanel>
+            <local:ControlExample.Options>
+                <Grid>
+                    <Grid.RowDefinitions>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                        <RowDefinition/>
+                    </Grid.RowDefinitions>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="Auto"/>
+                    </Grid.ColumnDefinitions>
+
+                    <TextBlock Text="Minimum:" Margin="0,0,0,0" Grid.Row="0" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MinimumValue" Grid.Row="0" Grid.Column="1" Margin="10,-5,0,0" Value="500" SpinButtonPlacementMode="Compact"/>
+
+                    <TextBlock Text="Maximum:" Margin="0,7,0,0" Grid.Row="1" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="MaximumValue" Grid.Row="1" Grid.Column="1" Margin="10,5,0,0" Value="1000" SpinButtonPlacementMode="Compact"/>
+
+                    <TextBlock Text="StepFrequency:" Margin="0,5,0,0" Grid.Row="2" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="StepFrequencyValue" Grid.Row="2" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact"/>
+
+                    <TextBlock Text="SmallChange:" Margin="0,5,0,0" Grid.Row="3" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="SmallChangeValue" Grid.Row="3" Grid.Column="1" Margin="10,5,0,0" Value="10" SpinButtonPlacementMode="Compact"/>
+
+                    <TextBlock Text="LargeChange:" Margin="0,5,0,0" Grid.Row="4" Grid.Column="0"/>
+                    <muxc:NumberBox x:Name="LargeChangeValue" Grid.Row="4" Grid.Column="1" Margin="10,5,0,0" Value="100" SpinButtonPlacementMode="Compact"/>
```

Comment:
A11y wise, this is not ideal. Either you should set the `AutomationProperties.LabeledBy` property on the NumberBoxes to the textblocks or set the `AutomationProperties.Name` property of these NumberBoxes. Alternatively, we could use the NumberBox's header and maybe update the layout into a two column layout where each column contains NumberBoxes.

Reply:
Resolved, but not sure how does it work tho. Adding `AutomationProperties.Name` seems like it makes the narrator speaks its name upon focus, while adding `AutomationProperties.LabeledBy` seems doing nothing.

Reply:
Dang it, I think this is an issue with NumberBox and how it handles UIA. Guess we have to fall back to using the AutomationProperties.Name property, sorry about the confusion.

Reply:
Should be good now.